packages/dht/README.md

```diff
@@ -1,17 +1,99 @@
-# @streamr/dht (WIP)
+# @streamr/dht
 
-An implementation of the Kademlia-based Streamr DHT used for the trackerless network.
+A connectionful Kademlia based P2P distributed hash table (DHT). Implements an in-memory key-value data store where multiple creators of of data can list values behind a single key. The library is entirely browser compatible.
 
-## Generating Protobuf code
+All communication between peers utilizes protobuf, transported with the help of the custom `@streamr/proto-rpc` library.
 
-Auto-generated protobuf/gRPC code is compiled during the build process.
-```bash
-  npm run build
+Connections in the DHT are established using WebSocket or WebRTC connections. The nodes will decide internally based on peers' connectivity information which connection types to use.
+
+The DHT also provides an interface for sending messages from peer to peer with the `DhtNode#send` function. The sent message is routed over the network to the target node's id.
```

Comment:
```suggestion
The DHT also provides an interface for sending messages from peer to peer with the `DhtNode#send` function. The sent message is routed over the network to the target node.
```

Could also also briefly describe how the network topology is formed? I.e. nodes which have closest kademlia distance are neighbors, and how `randomPeers` is related to that? Are we always connected to all neighbors, or other nodes too?

Reply:
Not necessarily important to detail that here. Technical details can be added in a separate section in the future